spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/LauncherService.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.dataflow.server.service;
+
+import org.springframework.cloud.dataflow.core.Launcher;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+
+/**
+ * Offers services to retrieve the launchers that are available for the
+ * Spring Cloud Data Flow instance.
+ *
+ * @author Glenn Renfro
+ */
+public interface LauncherService {
```

Comment:
I think it is now a good time to rename all the classes and other resources related to `Launcher` to `TaskLauncher` as they are meant for Task Launcher only. I think this could also avoid confusion from what we obtain from Skipper on stream deployer repository.

Reply:
Looking at the `Launcher` being the entity model that composes the `TaskLauncher`, I have to take my comment back above as we need to align with this model `Launcher`. All good.